src/box/lua/stat.c

```diff
@@ -271,6 +302,20 @@ lbox_stat_net_call(struct lua_State *L)
 	lua_rawset(L, -3);
 	lua_pop(L, 1);
 
+	lua_pushstring(L, "STREAMS");
+	lua_rawget(L, -2);
+	lua_pushstring(L, "current");
+	lua_pushnumber(L, iproto_stream_count());
+	lua_rawset(L, -3);
+	iproto_stream_rmean_foreach(set_stat_item, set_iproto_thread_request_stat_item, L);
+	lua_pushstring(L, "requests");
+	lua_rawget(L, -2);
+	lua_pushstring(L, "current");
+	lua_pushnumber(L, iproto_stream_request_count());
+	lua_rawset(L, -3);
+	lua_pop(L, 1);
+	lua_pop(L, 1);
+
```

Comment:
The docbot message describes a different format. In particular, 'requests' isn't documented. Please write the docbot request in such a way that even someone completely unfamiliar with the code can document the feature.